src/include/session.h

```diff
@@ -82,8 +83,8 @@ struct __wt_session_impl {
     /*
      * Each session keeps a cache of data handles. The set of handles can grow quite large so we
      * maintain both a simple list and a hash table of lists. The hash table key is based on a hash
-     * of the data handle's URI. The hash table list is kept in allocated memory that lives across
-     * session close - so it is declared further down.
+     * of the data handle's URI. The hash table list is kept in allocated memory until a session
+     * is closed, when it is freed. It is declared further down.
```

Comment:
This comment change is not correct, I think. Everything in the session after `flags` is not freed on close. Everything in the session structure after `WT_SESSION_CLEAR_SIZE` persists after a close. That was the specific point of the original form of the comment.

Reply:
Right - I'll clarify the comment, the elements are discarded and freed even though the hash array is kept.

Reply:
Fixed.